src/test/java/com/twilio/http/RequestTest.java

```diff
@@ -196,13 +205,13 @@ public void testRegionAndEdgeInUrl() throws MalformedURLException {
     }
 
     @Test
-    public void testRegionInConstructor() throws MalformedURLException {
-        final Request request = new Request(HttpMethod.GET, Domains.ACCOUNTS.toString(), "/path/to/something.json?foo=12.34", "region");
+    public void testRegionInConstructor() {
```

Comment:
Should we keep the old test, and also add this one? (testing with and without `#bar` in the URL)

Reply:
It does not make any meaningful difference to test with and without the fragment on the end, it just adds more code.

Reply:
It's a legitimate concern. Like, what if things only work when _both_ query params and fragments are used but fail when it's just one or the other. Not saying it doesn't work in those cases, but someone later may refactor the logic where it does in fact not work properly, but this test would still pass.

But, yes, in general, I'm against more test code that can be reduced. Test code is just like source code: it needs to be maintained.

Reply:
I agree, it's just in this case we [already have tests that include just query params](https://github.com/twilio/twilio-java/pull/558/files#diff-d249e028337d87ed2f4499346031488fR71). That said, I don't believe we have a test with no query params and just a fragment, but the way it's split, just query params vs just a fragment is treated equivalently.